xs-cli/README.md

```diff
@@ -0,0 +1,42 @@
+# XS CLI Docker File
+## Description
+To bring an XS application to a HANA XS Server you need to deploy the application with the xs command line tool. This Dockerfile can wrap the xs command line client and the resulting image is intended to run with our Jenkins pipeline library [project "Piper"][piper]. 
+
+## Requirements
+* General requirements can be found in the [repository readme][general]
+* An S-User for [SAP ONE][sapone]
+* Download XS command line client ```XS_CLIENT00P_<version>.ZIP``` for Linux on x86_64 from [SAP ONE][sapone]

```

Comment:
Please use XS consistently. I think it's used as "XS Command-Line Interface" or XS CLI. @SarahNoack can you please check the naming in this doc?